pkg/observ/stats.go

```diff
@@ -92,7 +92,15 @@ func registerStatsStackDriverExporter(projectID string) (func(), error) {
 // registerViews register stats which should be collected in Athens
 func registerViews() error {
 	const op errors.Op = "observ.registerViews"
-	if err := view.Register(ochttp.DefaultServerViews...); err != nil {
+	if err := view.Register(
```

Comment:
Is it possible to keep the defaults and append new views, otherwise we will introduce a breaking change

Sorry for the delay

Reply:
Explained in the PR summary, the only two default views that are skipped are not very useful for Athens. I doubt any one wants metrics from them. But sure, I can add them back if we want Athens to be backward-compatible

Reply:
yes please as we do not know if anyone has come to rely on them